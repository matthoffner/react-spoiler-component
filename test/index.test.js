import React from 'react';
import Spoiler from '../src/index';

const Loading = () => {
    return (
        <div id="test">Loading...</div>
    );
};

test('No prop type errors', () => {
    const component = <Spoiler path="./test/component.js" selector="test"><Loading /></Spoiler>;
    expect(component).toMatchSnapshot();
});