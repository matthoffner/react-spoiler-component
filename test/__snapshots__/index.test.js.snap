// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No prop type errors 1`] = `
<Spoiler
  path="./test/component.js"
  selector="test"
>
  <Loading />
</Spoiler>
`;
